iTunes Search/iTunes Search/Model/SearchResult.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SearchResult.swift
+//  iTunes Search
+//
+//  Created by Mitchell Budge on 5/14/19.
+//  Copyright © 2019 Mitchell Budge. All rights reserved.
+//
+
+import Foundation
+
+struct SearchResult: Codable {
```

Comment:
For some searches the JSON returns nil values and the app can crash or not show anything. If you do a search for "Batman" for example under the music section of the switch you will see its not working. So to let the decoder know to ignore nil values we can make these properties optional like this:

```
let title: String?
let creator: String?
```

Then the app will return results without the nil ones and work when you search for Batman.